pkg/objstore/s3/s3.go

```diff
@@ -81,6 +83,12 @@ func parseConfig(conf []byte) (Config, error) {
 	if config.PutUserMetadata == nil {
 		config.PutUserMetadata = make(map[string]string)
 	}
+
+	// Use the default minio minPartSize if not set
+	if config.PartSize == 0 {
+		config.PartSize = 1024 * 1024 * 128
```

Comment:
putting this `1024 * 1024 * 128` in some meaningfully named constant would be nice for a reader.

Reply:
hope it's better now, although I couldn't really find a meaningful variable name which wouldn't require an explanation anyway.